src/connector/src/parser/additional_columns.rs

```diff
@@ -44,21 +44,38 @@ pub static COMPATIBLE_ADDITIONAL_COLUMNS: LazyLock<HashMap<&'static str, HashSet
         HashMap::from([
             (
                 KAFKA_CONNECTOR,
-                HashSet::from(["key", "timestamp", "partition", "offset", "header"]),
+                HashSet::from([
+                    "key",
+                    "timestamp",
+                    "partition",
+                    "offset",
+                    "header",
+                    "payload",
+                ]),
             ),
             (
                 PULSAR_CONNECTOR,
-                HashSet::from(["key", "partition", "offset"]),
+                HashSet::from(["key", "partition", "offset", "payload"]),
             ),
             (
                 KINESIS_CONNECTOR,
-                HashSet::from(["key", "partition", "offset", "timestamp"]),
+                HashSet::from(["key", "partition", "offset", "timestamp", "payload"]),
+            ),
+            // remove s3 for no longer use
+            // (S3_CONNECTOR, HashSet::from(["file", "offset"])),
+            (
+                OPENDAL_S3_CONNECTOR,
```

Comment:
It seems currently we only have `include` tests for Kafka. Would you minding also adding a test for s3? Since s3 connector has quite some bugs, I'm a little worried about it.

Reply:
I'd merge this one first and do the test in following pr.

Reply:
https://github.com/risingwavelabs/risingwave/pull/18496